cmd/influx/v1_dbrp.go

```diff
@@ -0,0 +1,386 @@
+package main
+
+import (
+	"context"
+	"io"
+
+	"github.com/influxdata/influxdb/v2"
+	"github.com/influxdata/influxdb/v2/cmd/influx/internal"
+	"github.com/influxdata/influxdb/v2/dbrp"
+	"github.com/spf13/cobra"
+)
+
+func cmdV1DBRP(f *globalFlags, opt genericCLIOpts) *cobra.Command {
+	cmd := opt.newCmd("dbrp", nil, false)
+	cmd.Aliases = []string{"dbrp"}
+	cmd.Short = "Database retention policy mappings for v1 APIs"
+	cmd.Run = seeHelp
+
+	cmd.AddCommand(
+		v1DBRPCreateCmd(f, opt),
+		v1DBRPFindCmd(f, opt),
+		v1DBRPDeleteCmd(f, opt),
+		v1DBRPUpdateCmd(f, opt),
+	)
+
+	return cmd
+}
+
+var v1DBRPCRUDFlags struct {
+	id          string
+	json        bool
+	hideHeaders bool
+}
+
+var v1DBRPFindFlags struct {
+	BucketID string       // Specifies the bucket ID to filter on
+	DB       string       // Specifies the database to filter on
+	Default  bool         // Specifies filtering on default
+	ID       string       // Specifies the mapping ID to filter on
+	Org      organization // required  // Specifies the organization ID to filter on
+	RP       string       // Specifies the retention policy to filter on
```

Comment:
AFAICS the fields DB, Default, and RP aren't actually used in the implementation.